crates/autopilot/src/run_loop.rs

```diff
@@ -852,6 +888,7 @@ impl RunLoop {
     }
 }
 
+#[derive(Clone)]
```

Comment:
We can probably avoid that by providing only the required data to the `post_processing` and then the `select_winners` can own the `solutions` collection.

Reply:
Not sure how would that work since we call `select_winners` before `post_processing`.
But anyway I refactored `select_winners` to work with references so dropped Clone as well.